src/libraries/System.Net.Ping/tests/FunctionalTests/System.Net.Ping.Functional.Tests.csproj

```diff
@@ -29,5 +31,7 @@
              Link="Common\System\Net\NetworkInformation\UnixCommandLinePing.cs" />
     <Compile Include="$(CommonTestPath)System\Net\Capability.RawSocketPermissions.cs"
              Link="Common\System\Net\Capability.RawSocketPermissions.cs" />
+    <Compile Include="$(CommonPath)System\Text\ValueUtf8Converter.cs"
```

Comment:
Because `Interop.ReadLink` was being used by multiple other projects. The API signature was changed to now accept ROS<char> as the first argument, which allowed using a `ValueStringBuilder` for the construction of the string inside the while loop. And `DefaultPathBufferSize` is the constant that is used for the `stackalloc` passed to the `ValueStringBuilder` constructor.